JitsiMeetJS.js

```diff
@@ -57,6 +61,7 @@ var LibJitsiMeet = {
     },
     logLevels: Logger.levels,
     mediaDevices: JitsiMediaDevices,
+    environment: RTCBrowserType,
```

Comment:
@tsareg Why do you need this? We were trying to avoid this since long time.

Personally I prefer to keep simple the API of the library and I'm trying to avoid adding a lot of new methods to the interface.

Even if we need this I don't think that the browser detection is functionality that is related to the functionality of the lib-jitsi-meet. But if you absolutely need this we can extract it in separate npm package instead of adding it to the interface.